tests/parser.cpp

```diff
@@ -1745,7 +1745,8 @@ TEST(Parser, offsetof_type)
        "\n"
        "Program\n"
        " BEGIN\n"
-       "  offsetof: \n");
+       "  offsetof: \n"
```

Comment:
shouldn't `struct Foo` be somewhere here?

Reply:
was also curious, why this was missing - I think it's a separate issue I need to debug

Reply:
Fixed in the latest version.